bot/actions/deleteLogger.js

```diff
@@ -0,0 +1,19 @@
+const config = require('../../config.json');
+const Discord = require('discord.js');
+
+client.on("messageDelete", async msg => {
+    let logs = await msg.guild.fetchAuditLogs({type: 72});
+    let entry = logs.entries.first();
+  
+    let embed = new Discord.RichEmbed()
+      .setTitle("**REMOVED MESSAGE**")
+      .setColor("#0398fc")
+      .addField("Sent By", msg.author.tag, true)
+      .addField("Sent In Channel", msg.channel, true)
+      .addField("Deleted Message", msg.content)
+      .addField("Deleted By", entry.executor)
+      .setFooter(`Message ID: ${msg.id} | Sender ID: ${msg.author.id}`);
+  
+    let channel = msg.guild.channels.find(x => x.id === logchannel);
+    channel.send({embed});
+  });
```

Comment:
This should be in a file called `messageDelete.js`. See the `message.js` file for reference.

This is because the bot uses the file as the event handler.